tests/Doctrine/Tests/DBAL/Types/DateTimeImmutableTypeTest.php

```diff
@@ -17,7 +17,7 @@ class DateTimeImmutableTypeTest extends \PHPUnit\Framework\TestCase
     private $platform;
 
     /**
-     * @var DateTimeImmutableType
+     * @var Type
```

Comment:
Please revert.

Reply:
Should this parent/child issue be reported to upstream?

```
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/Doctrine/Tests/DBAL/Types/DateTimeImmutableTypeTest.php                                                                                              
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------- 
  26     Property Doctrine\Tests\DBAL\Types\DateTimeImmutableTypeTest::$type (Doctrine\DBAL\Types\DateTimeImmutableType) does not accept Doctrine\DBAL\Types\Type.  
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------- 
```

Reply:
No. Please try using `new DateTimeImmutableType()` instead of the factory.